autoload/codefmt.vim

```diff
@@ -446,8 +446,10 @@ function! codefmt#FormatBuffer(...) abort
     call maktaba#error#Shout('Error formatting file: %s', v:exception)
   endtry
 
-  let l:cmd = ":FormatCode " . l:formatter.name . "\<CR>"
-  silent! call repeat#set(l:cmd)
+  if a:repeat
+    let l:cmd = ":FormatCode " . l:formatter.name . "\<CR>"
```

Comment:
You could just put this part in a private function, like:
```vim
""
" @private
function! codefmt#FormatBufferAndSetRepeat(...) abort
  call call('codefmt#FormatBuffer', a:000)
  let l:cmd = ":FormatCode " . l:formatter.name . "\<CR>"
  silent! call repeat#set(l:cmd)
endfunction
```

Otherwise it's a weird signature for `codefmt#FormatBuffer`.